databases/week3/aditionalQueries.sql

```diff
@@ -0,0 +1,57 @@
+-- ----------------------Additional queries
+-- --------------------------------------------------------------------------------
+-- ------------------------------------Get meals that has a price smaller than a specific price fx 90
+
+SELECT *
+FROM Meal
+WHERE Mealsharing.Meal.price<90;
+
+-- ------------------------------------Get meals that still has available reservations
+
+Select SUM(Reservation.Number_of_guests) AS reserved_numbers, Meal.title, Meal.description, Meal.max_reservation
+from Reservation
+join Meal on Meal.idMeal=Reservation.meal_id
+GROUP BY Reservation.meal_id
+HAVING reserved_numbers<Meal.max_reservation;
+
```

Comment:
@hamedveloper , I wonder if a reservation haven't made on a meal, does the query still show the available reservations for that?